### ISSUE TITLE
OCS Operator expects NooBaa CRD to be present on the cluster

### ISSUE THREAD

**Issue by bindrad** [red-hat-storage/managed-fusion-agent#27]
We cannot directly install OCS-Operator because it expects NooBaa CRDs to be present on the cluster. Installing OCS-Operator does not bring NooBaa CRDs and OCS-Operator throws the error.

`{"level":"error","ts":1679650474.2685232,"logger":"controller-runtime.source","msg":"if kind is a CRD, it should be installed before calling Start","kind":"NooBaa.noobaa.io","error":"no matches for kind \"NooBaa\" in version \"noobaa.io/v1alpha1\"","stacktrace":"sigs.k8s.io/controller-runtime/pkg/source.(*Kind).Start.func1.1\n\t/remote-source/app/vendor/sigs.k8s.io/controller-runtime/pkg/source/source.go:139\nk8s.io/apimachinery/pkg/util/wait.runConditionWithCrashProtectionWithContext\n\t/remote-source/app/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:233\nk8s.io/apimachinery/pkg/util/wait.WaitForWithContext\n\t/remote-source/app/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:660\nk8s.io/apimachinery/pkg/util/wait.poll\n\t/remote-source/app/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:594\nk8s.io/apimachinery/pkg/util/wait.PollImmediateUntilWithContext\n\t/remote-source/app/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:545\nsigs.k8s.io/controller-runtime/pkg/source.(*Kind).Start.func1\n\t/remot...
`

This issue happens even after setting [reconcileStrategy of MCG](https://github.com/red-hat-storage/ocs-operator/blob/release-4.12/api/v1/storagecluster_types.go#L342) to ignore in storageCluster CR

**Comment by mulbc** [comment]
Could we install the CRD and then not have any NooBaa CR?

**Comment by bindrad** [comment]
> Could we install the CRD and then not have any NooBaa CR?

Yes, that is possible. As a workaround, we are going to do the same.
